example-auth/src/App.js

```diff
@@ -46,6 +47,25 @@ class App extends React.Component {
     }
   }
 
+  async register(email, password){
+    try {
+      // Set error to false so if we are successful the error doesn't perist making bad UX Design.
+      // also set the loading prop to true to signal to the user we are processing his request.
+      await this.setState({ error: false })
+
+      // Create the session, if this fails it will error and be caught by the catch(err).
```

Comment:
```suggestion
      // Create the account, if this fails it will error and be caught by the catch(err).
```

We use the `account.createSession` method for creating a new session 🙂